purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -104,6 +105,17 @@ class Purchases internal constructor(
             purchasesOrchestrator.updatedCustomerInfoListener = value
         }
 
+    /**
+     * The listener is responsible for handling actions that happened in the Customer Center.
+     * Make sure [removeCustomerCenterListener] is called when the listener needs to be destroyed.
+     */
+    var customerCenterListener: CustomerCenterListener?
```

Comment:
I wonder if we would want to support multiple listeners... But then again, this is the API we have for the `CustomerInfo`, so it might make more sense to be consistent 👍